src/ChangeStream.php

```diff
@@ -40,6 +40,10 @@ class ChangeStream implements Iterator
 
     const CURSOR_NOT_FOUND = 43;
 
+    /**
+     * @param MongoDB\Driver\Cursor $cursor
```

Comment:
No need to change this, but just a note: Doc block comments _can_ take advantage of the shorter class names (from `use` imports). It's just our rendered RST documentation that needs the FQCNs.